### DISCUSSION TITLE
Why would AO 7 open to the setup screen and then just close?

### DISCUSSION THREAD

**Discussion by zippy77777** [kubilus1/autoortho#424] [Q&A]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: [e.g. Windows 10]
- XPlane Version: [e.g. XP12]
- AutoOrtho Version 0.5.1

### question

Asking for a friend.
I cannot do anything coz the screen does not stay open long enought for me to click RUN.

**Comment by sergiolamares** [comment]
same here, I need to click more then once to keep it open

**Comment by zippy77777** [comment]
Mine is fine on Windows 11 and AO 7 but a friend of mine is on Windows 10 and can only use v5 of AO. The setup menu will NOT stay open.

**Reply by md11jock** [reply]
Exact same issue here. My Win 10 won't run anything past 5.1

**Comment by Jeff2122** [comment]
Check if there is a file called ".autoortho" in your "C:\Users\YOURNAME\" folder. If not, create an empty one and see if it will start. It should overwrite the contents once you press the "Save" button.

**Reply by zippy77777** [reply]
Thx for your reply. OK so no .autoortho file. I create a blank one. The opening screen is now there. I fill out the details and click save and it just sits on updating config and nothing happens. I look at the file and it's still blank. AO not saving.

**Reply by Jeff2122** [reply]
Here is a blank one created on my system. Plug it in place of yours and try that. Add the period and remove the .txt. 
I am not sure what else can cause this. Windows is fickle - it has surprised me too many times to be surprised anymore.
I wonder if you installed AO 0.7.0 as admin, or did you download the exec and run it that way?

[autoortho.txt](https://github.com/kubilus1/autoortho/files/13078572/autoortho.txt)

**Reply by Jeff2122** [reply]
@zippy77777 and @md11jock How are things going? Have you figured out anything yet? I am still feeling around too, but I cannot duplicate what you see here on my system.

**Reply by md11jock** [reply]
Now that I know how to get back to a working 5, I'm going to try 7 again but if the downloads are still slow, it will be awhile before I can report back.

**Reply by md11jock** [reply]
I have a custom scenery folder for each version and start each with a clean slate. Right now, I've gotten 7.0 to load and am waiting on the slow scenery downloads to see if it will run on a new copy of Xplane.

**Reply by zippy77777** [reply]
Guy I am trying to help hasn't got back to me :(
